topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -185,14 +185,16 @@ With the necessary background in place, you are ready to install Galaxy with Ans
 
 To proceed from here it is expected that:
 
-1. You have [Ansible installed](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html) on your local machine
-   > ### {% icon comment %} Comment: Running Ansible on your remote machine
-   > It is possible to have Ansible installed on the remote machine and run it there as well. You will need to update your inventory file to pass `ansible_connection=local`. Be **certain** that the playbook that you're building is stored somewhere safe like your user home directory. We will remove data at one point during this tutorial and would not want you to lose your progress.
+1. You have [Ansible installed](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html) on the machine where you will install Galaxy
+
+   > ### {% icon comment %} Comment: Running Ansible on remote machine
+   > It is possible to have Ansible installed on your laptop/local machine and run it against some remote hosts as well. We will *not* do that in this training.
    {: .comment}
+
 2. Your `ansible` version is `>=2.7`, you can check this by running `ansible --version`
 3. You have an [inventory file](../ansible/tutorial.html#inventory-file) with the VM or host specified where you will deploy Galaxy. We will refer to this group of hosts as "galaxyservers."
 4. Your VM has a public DNS name: this tutorial sets up SSL certificates from the start and as an integral part of the tutorial.
-5. Your VM has `python2.7` installed.
+5. Your VM has `python3` installed.
 6. In your inventory file, you have written the full DNS hostname that has been provided, and **not** `localhost`, as we will be requesting SSL certificates.
```

Comment:
Should we also add `ansible_python_interpreter = auto` so that ansible uses python 3 ?
```
[DEPRECATION WARNING]: Distribution Ubuntu 18.04 on host gcc-8.training.galaxyproject.eu should use /usr/bin/python3, but is using /usr/bin/python for
backward compatibility with prior Ansible releases. A future Ansible release will default to using the discovered platform python for this host. See
https://docs.ansible.com/ansible/2.9/reference_appendices/interpreter_discovery.html for more information. This feature will be removed in version 2.12.
Deprecation warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.
```

Reply:
If you're getting that error, it means that some step has installed py2. Which tutorials have you tested so far?

Reply:
I've been using that warning message as an indicator that something has gone wrong. But yes, +1 to setting ansible_python_interpreter

Reply:
I thought the VMs had a clean state (totally my fault ... 😆), so I followed the instructions in this tutorials that linked to the ansible docs which said to use apt from their ppa to install ansible, which did install python2